Gems/EMotionFX/Code/Source/Editor/SkeletonModelJointWidget.cpp

```diff
@@ -19,14 +25,25 @@
 
 namespace EMotionFX
 {
+    struct HLineLayout : public QVBoxLayout
+    {
+        HLineLayout(QWidget* parent = nullptr)
+        {
+            setContentsMargins(0, 5, 0, 5);
+            auto* frame = new QFrame{parent};
```

Comment:
Nitpick: Any reason you use QFrame{} instead of QFrame()?
Also personally I don't like seeing over using auto keyword. Not a big deal tho.